backend/src/main/java/com/google/step/servlets/WebhookServlet.java

```diff
@@ -78,8 +62,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
    */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    String advertiserKeyString = request.getParameter(ID_URL_PARAM);
+    if (advertiserKeyString == null) {
```

Comment:
do we also need to check for empty string?

Reply:
good idea, added the check